statServer/models/questions_model.ts

```diff
@@ -0,0 +1,88 @@
+/*
+El nuevo entrypoint tiene la siguiente estructura:
+
+GET /questions?since=2022-03-04
+
+{
+   "questionId": ".. ",  // Identificación de la pregunta 
+   "question": ".. ",  // Texto de la pregunta en español
+   "attempts":  #  // Número de veces que la pregunta se ha intentado 
+   "avgLatency": # // Tiempo promedio en segundo que la gente necesita para responder esta pregunta
+   "successProb": # //  La probabilidad de tener éxito respondiendo esta pregunta (valor entre cero y uno)
+}
+since: fecha para considerar los datos sólo después de la fecha dada, es opcional, si no está se consideran todos los datos disponibles.
+
+Para los textos de las preguntas se va a usar el resultado del issue Issue 54
+*/
+
+import { Questions } from '../types/questions';
+import { db } from "../db";
+import { RowDataPacket } from "mysql2";
+
+
+export const getQuetionsByDates = (callback: Function, dates: any) => {
+
+    let { since, to } = dates;
+    // console.log(since)
+    // console.log(to)
+
+    if (since === '' || to === '') {
+        console.log('cambio de since o to')
+        since = '01/01/0001'
+        to = '12/12/9999'
+    }
```

Comment:
Esta implementación falla:
http://localhost:8001/album_stats/questions?to=09/09/2023 
	debería retornar filtrando por to < 09/09/2023, pero retorna vacío 
http://localhost:8001/album_stats/questions?since=01/01/2020
	debería retornar filtrando por from > 01/01/2020, pero retorna vacío 
http://localhost:8001/album_stats/questions?since=&to=01/01/1900
	debería retornar vacío porque no hay datos antes de 01/01/1900, pero retorna todo
http://localhost:8001/album_stats/questions?to=&since=09/09/2023
	debería retornar vacío porque no hay datos después de 09/09/2023, pero retorna todo

Mucho mejor:
    since = (since ?? '1970-01-01') || '1970-01-01'
    to = (to ?? '3000-01-01') || '3000-01-01'

 Nótese que estoy proponiendo cambiar de %d/%m/%Y a  %Y-%d-%m para simplificar el código.

Reply:
/Se reliza el cambio de la fecha y las prubas segun las url dadas y la nueva estructura d elas fechas, donde se obitiene los datos deseado url probadas 
se re estructuro la fecha para consultas en formato yyy/mm/dd

http://localhost:8001/album_stats/questions?to=2023/09/09
filtrando por to < 2023/09/09, retorna los datos datos antes de esa fecha
http://localhost:8001/album_stats/questions?since=2020/01/01
filtrando por since> 2020/01/01, pero retorna todos los datos después de esa fecha
http://localhost:8001/album_stats/questions?since=&to=1900/01/01
filtrando por to < 01/01/1900, retorna vacío
http://localhost:8001/album_stats/questions?to=&since=2023/09/09
se realiza prueba y retorna vacío 

se establece la sugerencia brindada dentro del código